db.py

```diff
@@ -43,6 +43,13 @@ def addUser(aUserObject):
         cur = con.cursor()
         cur.execute(ins)
 
+def removeUser(aUserName):
+    con = sqlite3.connect(DB_FILE)
+    with con:
+        rem = "DELETE FROM users WHERE userName = '{}'".format(aUserName)
```

Comment:
This is susceptible to SQL injection attacks.  Use the [DB-API’s parameter substitution](https://docs.python.org/3/library/sqlite3.html) instead, e.g.
```python
cur.execute("insert into lang values (?, ?)", ("C", 1972))
```

Reply:
How is that any safer?